src/solvers/strings/string_refinement.h

```diff
@@ -45,7 +45,8 @@ Author: Alberto Griggio, alberto.griggio@gmail.com
   " --string-input-value st      restrict non-null strings to a fixed value st;\n" /* NOLINT(*) */ \
   "                              the switch can be used multiple times to give\n" /* NOLINT(*) */ \
   "                              several strings\n" /* NOLINT(*) */ \
-  " --max-nondet-string-length n bound the length of nondet (e.g. input) strings\n" /* NOLINT(*) */
+  " --max-nondet-string-length n bound the length of nondet (e.g. input) strings;\n" /* NOLINT(*) */ \
+  "                              set to 10000 by default\n" /* NOLINT(*) */
```

Comment:
Could we possibly have a constant or a macro for 10000 so that we don't have a problem with the comment and the setting of the value getting out of sync?

Reply:
Added a constant

Reply:
Thank you.